ui/app/routes/jobs/run/templates/template.js

```diff
@@ -0,0 +1,28 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import notifyForbidden from 'nomad-ui/utils/notify-forbidden';
+
+export default class RunRoute extends Route {
```

Comment:
Naming convention request: I think this is borrowed from routes/jobs/run and duplicated here (jobs/run/templates/template) as well as jobs/run/templates.

Could we use the more standard RunTemplatesRoute and RunTemplatesTemplateRoute for easier grepping?

Reply:
Will patch this when I throw the commit up for deleting inside of a row.